packages/compiler-core/src/parse.ts

```diff
@@ -445,6 +445,7 @@ function parseTag(
 
     if (tag === 'slot') tagType = ElementTypes.SLOT
     else if (tag === 'template') tagType = ElementTypes.TEMPLATE
+    else if (/[Pp]ortal/.test(tag)) tagType = ElementTypes.PORTAL
```

Comment:
```suggestion
    else if (tag === 'portal' || tag === 'Portal') tagType = ElementTypes.PORTAL
```

I think this would be much more performant.

Reply:
not only performance , previous one is not right  no `^` and `$`